microservices/nris_api/backend/.env-example

```diff
@@ -0,0 +1,21 @@
+export DB_HOST=postgres
+export DB_USER=
```

Comment:
Should these be blank? They aren't for MDS.

Reply:
It's the .env-example file, and those environment variables are used by the nris_api to connect to the mds_core postgres database with the nris user.